README.md

```diff
@@ -35,14 +35,22 @@ Currently, the supported hierarchy types are:
 - macroregion
 - neighbourhood
 - region
+- postalcodes (optional, see configuration)
 
 Other types may be included in the future.
 
 [This page](https://github.com/whosonfirst/whosonfirst-placetypes) has a description of all the types supported by Who's on First.
 
 ## Configuration
 
-The sole Pelias configuration option available is `imports.whosonfirst.datapath`.  It should point to the root of where Who's On First data has been downloaded.
+This importer is configured using the [`pelias-config`](https://github.com/pelias/config) module. 
+The following configuration options are supported by this importer.
+
+| key | required | default | description |
+| --- | --- | --- | --- |
+| `imports.whosonfirst.datapath` | yes | | full path to the root of where `Who's On First` data has been downloaded |
```

Comment:
Good idea, something like this would be great:

full path to where Who's on First data is located (note: the included downloader script(make this a link to the "downloading the data section") will automatically place the WOF data here, and is the recommended way to obtain WOF data)